xbmc/FileItem.cpp

```diff
@@ -3578,11 +3578,44 @@ std::string CFileItem::GetBaseMoviePath(bool bUseFolderNames) const
 
       URIUtils::GetParentPath(name2, strMovieName);
     }
+
+    // Remove 'Disc' subdirectory so actual title passed to scraper
+    if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+            CSettings::SETTING_MYVIDEOS_HANDLEDISCN))
+    {
+      size_t iDisc = GetDiscPath(strMovieName);
+      if (iDisc)
+        strMovieName.erase(iDisc + 1);
+    }
   }
 
   return strMovieName;
 }
 
+size_t CFileItem::GetDiscPath(std::string strPath) const
+{
+  // Handle both Disc and Disk spellings
+  size_t iPos = strPath.rfind("/Disc ");
+  if (iPos == std::string::npos)
+    iPos = strPath.rfind("/Disk ");
```

Comment:
Why exactly this two spellings? What about other languages?

Please remove the type prefix from the variable's name (coding guideline).

Reply:
I agree. I wasn't sure how to tackle this. Can you refer to strings.po in code??

Reply:
I don't think that strings.po is correct way of doing it. I for example have different kodi clients working with the same shared files, but have different language settings. Maybe we just stick to one string, either "Disc" or "Disk". I'm not a native speaker, so I do not know which one is more common.

Reply:
Yes, of course you can.  See https://github.com/xbmc/xbmc/blob/master/xbmc/music/MusicDatabase.cpp#L896 for how the music db localises the "disc" string.  

@ksooo Either english spelling is actually correct, although there could be a semantic discussion about whether one spelling refers more to a HDD than an optical disc.  I'd suggest strings.po is the correct place and the code checks for both English (and translated) versions so as to catch everything (hopefully!).

Reply:
@ksooo Thank you for all your comments. I hope I've made all the adjustments correctly.
@the-black-eagle Thank you. I will look at that.